code_samples/theme_basics/README.md

```diff
@@ -0,0 +1,41 @@
+# Theme Basics [![JetBrains IntelliJ Platform SDK Docs](https://jb.gg/badges/docs.svg)][docs]
+*Reference: [Creating Custom UI Themes in IntelliJ SDK Docs][docs:themes]*
+
+## Quickstart
+
+Custom UI Themes are available beginning in version 2019.1.
+
+Creating a custom UI Theme is a process of choosing a base IDE Theme (Light or Darcula) then changing aspect
+of the base Theme definition. Custom UI Themes can:
```

Comment:
Suggest: _Creating a custom UI Theme is a process of choosing a base IDE Theme (Light or Darcula,) then changing aspects of the base Theme definition._

Reply:
Is this comma: `(Light or Darcula,) then` written by purpose?